docs/getting-started.md

```diff
@@ -90,6 +90,17 @@ Using project "myproject".
 
 Now we can move on to creating our application using `odo`.
 
+
+#### Notes with regards to Service Catalog / `odo service` functionality:
+
+In order to make **full use** of all Odo functionality, it's recommended to enable [service catalog](https://docs.openshift.com/container-platform/3.6/architecture/service_catalog/index.html) with OpenShift.
```

Comment:
3.6 is really really old version. It would be better to refer to latest stable - 3.11

Reply:
it looks like you can use https://docs.openshift.com/container-platform/latest/architecture/service_catalog/index.html that will redirect to the latest version automaticaly

Reply:
:+1: